app/controllers/api/v2/ansible_inventories_controller.rb

```diff
@@ -90,7 +90,7 @@ def action_permission
       private
 
       def schedule_params
-        template_name = Setting::Ansible.find_by(:name => 'ansible_inventory_template').value
+        template_name = Setting.find_by(:name => 'ansible_inventory_template').value
```

Comment:
we need to do the following instead, otherwise we would not ge the default value (as it is no longer in the DB)
```suggestion
        template_name = Setting['ansible_inventory_template']
```